src/components/track/mobile/TrackTile.tsx

```diff
@@ -63,6 +64,29 @@ const formatCoSign = ({
   return messages.reposted
 }
 
+export const RankIcon = ({
+  showCrown,
+  index,
+  className,
+  isVisible = true
+}: {
+  showCrown: boolean
+  index: number
+  isVisible?: boolean
+  className?: string
+}) => {
+  return (
+    <>
+      {isVisible && (
```

Comment:
this is an interesting one to review for our little style guide. i know we had said x && <component>, but i think ternary with null might be more performant than this. no need for action on this PR, just somethin i thot of

Reply:
Interesting - why would ternary with null be more performant?